gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1256,8 +1392,6 @@ def main(args):
         # NOTE: The following step is not yet implemented for joint and should not
         # be reviewed for it
         if args.export_vcf:
-            if data_type == "joint":
-                raise ValueError("Joint data type is not yet supported for VCF export.")
             if contig and test:
                 raise ValueError(
                     "Test argument cannot be used with contig argument as test filters"
```

Comment:
Please update this statement now that we dropped the old test filtering

Reply:
@KoalaQin , small thing here

Reply:
I just added here too.